externs/es6.js

```diff
@@ -394,6 +394,23 @@ function Int8Array(length, opt_byteOffset, opt_length) {}
 /** @type {number} */
 Int8Array.BYTES_PER_ELEMENT;
 
+/** 
+* @param {Array<number>} source
```

Comment:
It does need to change, just sure to what. 

There are two call scenarios, one with a single argument, which would have to be something iterable returning numbers.

Second is with a source and map function, then the source can by anything iterable as the mapFn provides with numbers. 

Eg. Uint8Array.from(['a', 'aa', 'aaa'], v => v.length)